src/usrp_drivers/utils/driveroptions.cpp

```diff
@@ -428,46 +477,6 @@ double DriverOptions::get_agc_signal_read_delay() const
     return agc_signal_read_delay_;
 }
 
-/**
- * @brief      Gets all antennas connected to N200s
- *
- * @return     The list of antennas connected to N200s
- */
-std::vector<size_t> DriverOptions::get_main_antennas() const
-{
-    return main_antennas_;
-}
```

Comment:
I guess these deleted parameters weren't used in the c++ code?

Reply:
Nope they weren't, I figured there was no point keeping them around. We don't parse the entire options file anyways